src/ansible_navigator/runner/command.py

```diff
@@ -0,0 +1,21 @@
+""" Herewithin lies the ability to have ansible-runner
+run a command in a synchronous manner
+"""
+from typing import Tuple
+
+from ansible_runner import run_command
+
+from .command_base import CommandBase
+
+
+class Command(CommandBase):
+    # pylint: disable=too-few-public-methods
+    # pylint: disable=too-many-instance-attributes
+    """a runner wrapper"""
+
+    def run(self) -> Tuple[str, str, int]:
+        """run"""
+
+        self.generate_run_command_args()
+        out, err, ret_code = run_command(**self._runner_args)
+        return out, err, ret_code
```

Comment:
```suggestion
        return run_command(**self._runner_args)
```

Reply:
I prefer it how it is b/c it's easier to put in a break point and introspect the values